drivers/hmis/app/graphql/mutations/publish_form_definition.rb

```diff
@@ -8,7 +8,7 @@ module Mutations
   class PublishFormDefinition < CleanBaseMutation
     argument :id, ID, required: true
 
-    field :form_identifier, Types::Forms::FormIdentifier, null: false
+    field :form_identifier, Types::Forms::FormIdentifier, null: true
```

Comment:
Fixes https://green-river.sentry.io/issues/5615330060/?project=4504006381273088&query=&referrer=issue-stream&statsPeriod=1h&stream_index=0

Reply:
Hm, should we add a `field :errors` which seems to be our pattern elsewhere?

Reply:
Oh, I see this has been added by the frontend schema. But don't quite understand how if it was not added here? https://github.com/greenriver/hmis-frontend/pull/851/files#diff-e38721dd872401d491c291485f2e562aac599fcdf386e9284d0e424b38d1c662R59-R61

Reply:
parent class `CleanBaseMutation` sets the `errors` field!